.gitignore

```diff
@@ -6,3 +6,6 @@ Tests/App/cache/*
 !Tests/App/cache/.gitkeep
 Tests/App/web/*
 !Tests/App/web/uploads/.gitkeep
+
+.DS_Store
```

Comment:
Should not be present in this file. You must declare common ignored files in your [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)